stablehlo/reference/Element.h

```diff
@@ -27,7 +30,10 @@ class Element {
  public:
```

Comment:
Not directly related to this CL, but classes should be documented IMO

Reply:
Point noted! Will make sure to document the classes `Tensor, Element, IndexSpaceIterator`.

Reply:
done.